src/serviceFacade/analysesServiceFacade.js

```diff
@@ -0,0 +1,11 @@
+import callApi from "../common/callApi";
+
+function getAnalyses(key, { rowsPerPage, orderBy, order, page, filter }) {
```

Comment:
I think we're chopping off `ServiceFacade` from these filenames now, so this file should just be `analyses.js`